unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SupportAnalytics/SupportAnalytics.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SupportAnalytics
+{
+    public class SupportAnalytics : ISupportAnalytics
+    {
+        private const string PLAYER_OPENED_SUPPORT = "player_opened_support";
+
+        public static SupportAnalytics i { get; private set; }
+
+        private readonly IAnalytics analytics;
+
+        public SupportAnalytics(IAnalytics analytics)
+        {
+            this.analytics = analytics;
+            i ??= this;
+        }
```

Comment:
Why are you doing it Singelton?

Reply:
followed the stucture of other analytics as well

Reply:
I suppose that other analytics is a Legacy when Singeltons where everywhere in the project. Now we are shifting to the Dependency Injection and serviceLocator. If it is not used as a Singelton by anybody now, I would strongly recommend to remove singelton related code (again - YAGNI)